inspirehep/records/cli.py

```diff
@@ -19,7 +19,7 @@
 from inspirehep.records.indexer.cli import get_query_records_to_index, next_batch
 from inspirehep.records.tasks import batch_recalculate
 
-logger = logging.getLogger()
+LOGGER = logging.getLogger()
```

Comment:
this should be like the following (https://docs.python.org/3/library/logging.html#logger-objects) everywhere

```python
LOGGER = logging.getLogger(__name__)
```

Reply:
Yes, I know. I missed it. Sorry.